Bloxstrap/Resources/Strings.resx

```diff
@@ -761,6 +761,12 @@ Selecting 'No' will ignore this warning and continue installation.</value>
   <data name="Menu.Integrations.AllowActivityJoining.Title" xml:space="preserve">
     <value>Allow activity joining</value>
   </data>
+  <data name="Menu.Integrations.ShowAccountOnProfile.Description" xml:space="preserve">
+    <value>Shows what Roblox account your using on your Discord profile.</value>
```

Comment:
I suggest changing this to: "Show which Roblox account you are using on your Discord profile."

Reply:
do you think the original sounds weird or why do you think i should change it? 👍

Reply:
I believe replacing "what" with "which" sounds better. The part "Roblox account **your** using" is incorrect, it should be "Roblox account **you're** using". I used the uncontracted form of "you're" as I heard it's better for translation accuracy.

Reply:
thanks for the feedback, i've changed it. english is not my native language.

Reply:
@Flikter You’re is “you are.” Your is a possessive adjective, meaning it shows ownership.